docs/IE_DG/supported_plugins/HETERO.md

```diff
@@ -30,6 +30,8 @@ Another way to annotate a network is to set affinity manually using <code>ngraph
 
 @snippet openvino/docs/snippets/HETERO0.cpp part0
 
+> **NOTE**: If you set affinity manually, be careful at the current moment Inference Engine plugins don't support constant (`Constant`->`Result`) and empty (`Parameter`->`Result`) networks. Please avoid such subgraphs when you set affinity manually.
+
 The fallback policy does not work if even one layer has an initialized affinity. The sequence should be calling of automating affinity settings and then fix manually.
```

Comment:
I think logically this note should be after paragraph  about all layers should have affinity

Reply:
Fixed